pippo-core/src/main/java/ro/pippo/core/Response.java

```diff
@@ -26,13 +26,7 @@
 
 import javax.servlet.http.Cookie;
 import javax.servlet.http.HttpServletResponse;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.StringWriter;
+import java.io.*;
```

Comment:
I prefer to not use wildcard import. Can you please fix this line? If you use IntelliJ Idea you can take a look at [this](http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports).
Thanks